Pod/Classes/OTNoop.m

```diff
@@ -24,7 +24,9 @@ @implementation OTNoopSpan
 - (id<OTTracer>)tracer { return g_defaultNoopTracer; }
 - (id<OTSpanContext>)context { return g_defaultNoopSpanContext; }
 - (void)setOperationName:(NSString*)operationName {}
-- (void)setTag:(NSString*)key value:(NSString*)value {}
+- (void)setTag:(NSString*)key stringValue:(nonnull NSString *)value {}
```

Comment:
I see your point, yeah. It's unfortunate that this ends up being a breaking change... I suppose we should honor semver and bump the major version number accordingly.